test/integration/search/search_test.rb

```diff
@@ -150,97 +150,121 @@ def test_only_contains_fields_which_are_present
     assert_equal [{ "slug" => "farming" }], results[1]["specialist_sectors"]
   end
 
-  def test_facet_counting
+  def test_aggregate_counting
     populate_content_indexes(section_count: 2)
 
-    get "/search?q=important&facet_mainstream_browse_pages=2"
+    get "/search?q=important&aggregate_mainstream_browse_pages=2"
 
     assert_equal 4, parsed_response["total"]
 
-    facets = parsed_response["facets"]
+    aggregate = parsed_response["aggregate"]
 
     assert_equal({
       "mainstream_browse_pages" => {
         "options" => [
-          { "value" => { "slug" => "1" }, "documents" => 2 },
-          { "value" => { "slug" => "2" }, "documents" => 2 },
+          { "value" => { "slug" => "browse/page/1" }, "documents" => 2 },
+          { "value" => { "slug" => "browse/page/2" }, "documents" => 2 },
         ],
         "documents_with_no_value" => 0,
         "total_options" => 2,
         "missing_options" => 0,
         "scope" => "exclude_field_filter",
       }
-    }, facets)
+    }, aggregate)
+  end
+
+  # backwards compatibility
```

Comment:
Can you elaborate on this comment (eg "we changed facet -> aggregate but still support both)

Also: This method has the name same as the above one. Assuming it's meant to be `test_facet_counting`!